cfme/storage/volume.py

```diff
@@ -293,6 +316,16 @@ def backups(self):
         view = navigate_to(self, 'Details')
         return int(view.entities.relationships.get_text_of('Cloud Volume Backups'))
 
+    @property
+    def snapshots(self):
```

Comment:
**REQUIRED**: Wrong property naming. One would expect this to be a link to snapshot collection for this volume. Rename this to ``snapshot_count`` or something like that.